frontend/__tests__/components/command-line-tools.spec.tsx

```diff
@@ -0,0 +1,79 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { CommandLineTools } from '../../public/components/command-line-tools';
+
+describe('CommandLineTools', () => {
+  let wrapper;
+  const obj = {
+    data: [
+      {
+        apiVersion: 'console.openshift.io/v1',
```

Comment:
not: mind removing the extra whitespace?

Reply:
(between lines)

Reply:
I think this was from the linter, but it's now gone since the object is smaller.